poetry/packages/locker.py

```diff
@@ -301,5 +304,7 @@ def _dump_package(self, package):  # type: (poetry.packages.Package) -> dict
             }
             if package.source_type:
                 data["source"]["type"] = package.source_type
+            if package.source_type == "directory":
+                data["source"]["develop"] = package.develop
```

Comment:
Is this the right place to put this? It's an attribute of the package rather than the source, so I think it's better to put it in the `[[package]]` section.

Reply:
Good point! Changed it.